consensus/module.go

```diff
@@ -172,8 +172,9 @@ func (m *consensusModule) loadPersistedState() error {
 	}
 
 	// TODO: Populate the rest of the state from the persistence module: validator set, quorum cert, last block hash, etc...
-	m.Height = uint64(latestHeight) + 1 // +1 because the height of the consensus module is where it is actively participating in consensus
-	m.appHash = string(appHash)
+	// m.Height = uint64(latestHeight) + 1 // +1 because the height of the consensus module is where it is actively participating in consensus
```

Comment:
Why is this left as comments... 

What's going on with the appHash? Just logging?

Reply:
Adding back w/ previous state hydration.